powerauth-admin/src/main/java/io/getlime/security/app/admin/controller/ApplicationController.java

```diff
@@ -157,7 +160,7 @@ public String applicationVersionCreate(@PathVariable("applicationId") String id,
      * @param model Model with passed parameters.
      * @return "callbackCreate" view.
      */
-    @GetMapping("/application/detail/{applicationId}/callback/create")
+    @GetMapping("/application/detail/{applicationId}/callback/create/form")
```

Comment:
Changing URL is not backward compatible but for Admin API we may afford that.

Reply:
Yes, I think we can assume those are just called internally.